README.md

```diff
@@ -48,6 +48,22 @@ First time contributors can read [CONTRIBUTION.md](https://github.com/immadisair
 
 7. If the app was built successfully, you can test it by running it on either a real device or an emulated one by going to _Run>Run 'app'_ or pressing the Run icon in the toolbar.
 
+### Screenshots 
+<table>
+  <tr>
+    <td><img src="https://user-images.githubusercontent.com/51189244/60667450-321c4d80-9e87-11e9-84c9-d14485a39bda.jpeg" height = "480" width="270"></td>
+    <td><img src="https://user-images.githubusercontent.com/51189244/60667464-37799800-9e87-11e9-986e-c3c2a936b42a.jpeg" height = "480" width="270"></td>
+    <td><img src="https://user-images.githubusercontent.com/51189244/60667460-35173e00-9e87-11e9-9127-5a0e6fac9343.jpeg"" height = "480" width="270"></td>
+  </tr>
```

Comment:
I guess there is no need of having the same kind of pictures again. Can you please remove this.